README.md

```diff
@@ -175,3 +175,10 @@ for more information.
 [cloud-video]: https://cloud.google.com/video-intelligence/
 [cloud-vision]: https://cloud.google.com/vision
 [cloud-webrisk]: https://cloud.google.com/web-risk/
+
+## Links
+
+- [Go on Google](https://cloud.google.com/go/home)
+- [Getting started with Go](https://cloud.google.com/go/getting-started)
+- [App Engine Quickstart](https://cloud.google.com/appengine/docs/standard/go/quickstart)
```

Comment:
Note that the links in this README will show up on every page that corresponds to the cloud.google.com/go module. It might make more sense to have this link on a specific package page related to AppEngine. You can do that by putting a link in the package documentation if you want to do that (see golang/go#42968).

Reply:
There aren't really AppEngine (or Cloud Functions/Cloud Run)-specific packages. So, I think these make sense.